guides/v2.1/cloud/env/variables-deploy.md

```diff
@@ -106,6 +106,25 @@ stage:
       _merge: true
 ```
 
+### `ENABLE_GOOGLE_ANALYTICS`
+
+-  **Default**—_false_
+-  **Version**—Magento 2.1.4 and later
+
+```yaml
+stage:
+  deploy:
+   ENABLE_GOOGLE_ANALYTICS: false 
```

Comment:
Need to indent more so that users can copy it correctly for their YAML.

Reply:
The indenting is inconsistent throughout the file. I'll fix that.

Reply:
That is not exactly true. It should be two spaces or four spaces. The only other one I see is Queue configurations that has an indentation inconsistency. In any case, this is a review and that is just my observation.